hopsworks-web/yo/app/views/jobs.html

```diff
@@ -467,64 +467,86 @@ <h5 class="text-muted pull-right"> Number of executions {{jobsCtrl.getNumOfExecu
       </div>
     </div>
     <!-- /#wrapper -->
-  </div>
-  <tour class="ng-scope" step="projectCtrl.tourService.currentStep_TourThree"
-        post-tour="closeTour()"
-        ng-if="(projectCtrl.loadedView && projectCtrl.tourService.activeTour === 'spark' ||
+    <tour class="ng-scope" step="projectCtrl.tourService.currentStep_TourThree"
+          post-tour="closeTour()"
+          ng-if="(projectCtrl.loadedView && projectCtrl.tourService.activeTour === 'spark' ||
                   projectCtrl.loadedView && projectCtrl.tourService.activeTour === 'tensorflow')
           && (projectCtrl.tourService.informAndTips
               || projectCtrl.tourService.tipsOnly)
           ">
-    <virtual-step
-      class="ng-scope"
-      tourtip="Click Next OR click on the 'New job' button to continue creating your {{projectCtrl.tourService.activeTour}} job!"
-      tourtip-step="0"
-      tourtip-element="#buttonCreateJob"
-      tourtip-placement="bottom"
-      tourtip-next-label="Next"
-      on-proceed="jobsCtrl.newJob()"
-      tourtip-offset-vertical="-15"
-      tourtip-offset-horizontal="35"
+      <virtual-step
```

Comment:
@Limmen Before we refactor the html and js files. it's better to remove the new tabs to avoid conflicts in other people's PRs.

Reply:
Done